package.json

```diff
@@ -30,7 +30,8 @@
           "marc-records-editor.item.get",
           "inventory.instances.item.get",
           "instance-authority-links.instances.collection.get",
-          "instance-authority.linking-rules.collection.get"
+          "instance-authority.linking-rules.collection.get",
+          "mapping-rules.get"
```

Comment:
Please move it to `quickMARC: View, edit MARC authorities record` permission because the ticket is related to authority records

Reply:
I've already did - and it's merged and deployed and  it seems that this ticket is not necessary at all.